setup.py

```diff
@@ -9,8 +9,16 @@
 here = os.path.abspath(os.path.dirname(__file__))
 about = {}
 
-with open(os.path.join(here, "requirements.txt"), "r") as f:
-    requirements = f.read().strip().split("\n")
+requirements = ['SQLAlchemy==1.3.23',
+                'apache-airflow-providers-apache-beam==4.0.0',
+                'apache-airflow-providers-common-sql==1.0.0',
+                'apache-airflow-providers-facebook==3.0.1',
```

Comment:
How come the "apache-airflow-providers-*" are required when packaging, but not required in the main "requirments.txt" file

Reply:
i did change that to put them in requirements.txt instead of install_requires